cmd/root.go

```diff
@@ -79,8 +79,13 @@ func DefaultCommandRunFunction(services ...KoolService) CobraRunFN {
 			service.SetErrStream(cmd.ErrOrStderr())
 
 			if err := service.Execute(args); err != nil {
-				service.Error(err)
-				service.Exit(1)
+				if err.Error() == shell.ErrPromptSelectInterrupted.Error() {
```

Comment:
we should use more `errors.Is(err, targetErr)` for this type of checking

Reply:
other thing - for this to on the root command, I guess it should be more generic... maybe a new interface for `UserCancelled` implementing error coming from anywhere (in this case, answering No to the proposed fix).